src/tslint-cli.ts

```diff
@@ -153,11 +153,24 @@ const processFile = (file: string) => {
         process.exit(1);
     }
 
+    const rulesDirectories: Array<string> = [];
+    if (configuration.rulesDirectory) {
+        if (typeof configuration.rulesDirectory === "string") {
+            rulesDirectories.push(configuration.rulesDirectory);
+        } else {
+            rulesDirectories.push(...configuration.rulesDirectory);
+        }
+    }
+
+    if (argv.r) {
```

Comment:
Done in commit https://github.com/palantir/tslint/commit/ee60a0cd1b858bd9555f21e9b4efdb501ebb570d.